### ISSUE TITLE
this.TD is undefined in custom editor when using fixedColumnsLeft and viewportColumnRenderingOffset together

### ISSUE THREAD

**Issue by maouida** [handsontable/handsontable#6043]
### Description
When implementing the `prepare` method in a custom editor, the `td` argument and `this.TD` properties are `undefined`. This happens only when the spreadsheet uses `fixedColumnsLeft` and `viewportColumnRenderingOffset` settings together.

### Steps to reproduce
1. Open this pen: https://codepen.io/maouida/pen/xNOQXv?editors=1010
2. Click any cell on the first column which should trigger the `prepare` method.
3. Both `this.TD` and `td` parameters are `undefined`.
4. If the setting `viewportColumnRenderingOffset `is removed everything works as expected.

### Demo
https://codepen.io/maouida/pen/xNOQXv?editors=1010

### Your environment
* Handsontable version: 6.2.2
* Browser Name and version: Chrome
* Operating System: Windows

**Comment by AMBudnik** [comment]
I don't know how often you'd want to use `0` but you're right. If we delete it or use any other value it works.

Thank you for sharing @maouida that's a good catch.

**Comment by maouida** [comment]
@AMBudnik Actually any value that is below the value of `fixedColumnsLeft` will cause the same issue.
So for this will not work:
```
{
 fixedColumnsLeft: 2,
 viewportColumnRenderingOffset: 1
}
```
and this works fine:
```
{
 fixedColumnsLeft: 2,
 viewportColumnRenderingOffset: 2
}
```

**Comment by AMBudnik** [comment]
Thank you for doing the investigation. That's a good start!

**Comment by warpech** [comment]
The problem is still reproducible with 7.1.0, after some adjustments to the config: https://codepen.io/warpech/pen/gVvBLG

A small difference is that `this.TD` is now `null` instead of `undefined`

I am working on a fix

**Comment by aninde** [comment]
@maouida this bug will be fixed in `7.1.2`. The new version will be released next week.
It works in both cases, when 
```
{
 fixedColumnsLeft: 1,
 viewportColumnRenderingOffset: 0
viewportRowRenderingOffset: 0,
}
```
and
```
{
 fixedColumnsLeft: 2,
 viewportColumnRenderingOffset: 1
}
```
**Fixed result  in `7.1.2`**
Clicking in the first cell returns: 
`<td class='current highlight' style='height: 22px;'>2017</td>`

Demo: 
v. `7.1.2` https://codepen.io/Demytruk/pen/yLLYJYz
Chrome 77
MacOs Pro Catalina 10.15

**Comment by AMBudnik** [comment]
I'm happy to see this issue solved. Marwan please update to 7.2.0.
